nix/shell.sh

```diff
@@ -9,6 +9,21 @@ banner()
     echo "+---------------------------------------------------------+"
 }
 
+function libcrypto_alias {
+    if [[ -f $2 ]]; then
```

Comment:
I have trouble reading bash a lot of the times, so maybe we could add a comments explaining what the two arguments are? Or sometimes just setting them to named variables is sufficient
```
libcrypto_name = $1
libcrypto_binary_path = $2
alias libcrypto_name=libcrypto_binary_path
```